src/main/java/com/minecolonies/coremod/achievements/MineColoniesAchievement.java

```diff
@@ -53,8 +51,8 @@ public MineColoniesAchievement(final String id, final int offsetX, final int off
      * @param itemStackIcon The icon to show
      * @param parent        the parent achievement
      */
-    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final ItemStack itemStackIcon, final Achievement parent)
+    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final ItemStack itemStackIcon, final Advancement parent)
     {
-        super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, itemStackIcon, parent);
+        //super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, itemStackIcon, parent);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)